src/guides/parachain_deployment.md

```diff
@@ -82,13 +82,13 @@ Secret Key URI `//Alice` is account:
   SS58 Address:      5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
 ```
 
-## Reserve a ParaId on Rococo
+## Reserve a ParaId on Paseo
 
 Note: although it is possible to use specific UIs for registering your parachain, this guide only documents how to do it by submitting extrinsics directly through the Polkadot.js Console.
 
-To get reserve a ParaId for your parachain on Rococo, navigate to the [Polkadot.js Apps interface](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frococo-rpc.polkadot.io).
+To get reserve a ParaId for your parachain on Paseo, navigate to the [Polkadot.js Apps interface](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frococo-rpc.polkadot.io).
```

Comment:
Missed a link here

Reply:
Thank you for checking, fixed :heavy_check_mark: